tools/analyzer/client.py

```
@@ -1,26 +1,31 @@
+"""
 
 from __future__ import annotations
 
-import json
 import logging
-import re
-import time
-from typing import TYPE_CHECKING, Any, NoReturn, Protocol
+from typing import TYPE_CHECKING, Any
 
 import google.api_core.exceptions
```

Pylint (3.3.7)

[error] No name 'api_core' in module 'google'

(E0611)